pootle/templates/browser/_actions.html

```diff
@@ -1,5 +1,6 @@
 {% load i18n %}
 <ul>
+  {% if not can_suggest_only %}
```

Comment:
This should be displayed if user can review or can translate, but we don't need to hide this from users with suggest right.